homework-submission/todoapp/src/Stores/TodoStore.js

```diff
@@ -0,0 +1,53 @@
+import { observable, action } from 'mobx';
+import todoObj from '../data/todos.json';
+
+
+class TodosStore {
+  @observable todos = todoObj;
+
+  @action checkBoxHandler = (id) => {
+    const newTodos = this.todos.map(item => {
+      if (item.id === parseInt(id, 10)) {
+        return {
+          ...item,
+          done: !item.done
+        };
+      }
+      return item;
+    });
+    this.todos = newTodos;
+  }
+
+  @action deleteHandler = (ID) => {
+    const newTodos = this.todos.filter(item => item.id !== ID);
+    this.todos = newTodos;
+  }
+
+  @action editHandler = (ID) => {
+    const Index = this.todos.findIndex(todo => todo.id === ID)
```

Comment:
starting with a capital letter is referring to a constructor while this is not, So for a convention what about `index`?
```suggestion
    const index = this.todos.findIndex(todo => todo.id === ID)
```